model/src/test/kotlin/OrtResultTest.kt

```diff
@@ -146,29 +146,26 @@ class OrtResultTest : WordSpec({
     "getOpenIssues" should {
         "omit resolved issues" {
             val ortResult = OrtResult.EMPTY.copy(
-                repository = Repository.EMPTY.copy(
-                    config = RepositoryConfiguration(
-                        resolutions = Resolutions(
-                            issues = listOf(
-                                IssueResolution(
-                                    "Issue message to resolve",
-                                    IssueResolutionReason.CANT_FIX_ISSUE,
-                                    "comment"
-                                )
-                            )
-                        )
-                    )
-                ),
                 analyzer = AnalyzerRun.EMPTY.copy(
                     result = AnalyzerResult(
                         projects = emptySet(),
                         packages = emptySet(),
                         issues = mapOf(
-                            Identifier("Maven:org.oss-review-toolkit:example:1.0") to
-                                listOf(
-                                    Issue(message = "Issue message to resolve", source = ""),
-                                    Issue(message = "Non-resolved issue", source = "")
-                                )
+                            Identifier("Maven:org.oss-review-toolkit:example:1.0") to listOf(
```

Comment:
Just for my understanding: Is this change necessary because the original resolution from the repo config was not actually "baked" into the resolutions from the resolved configuration in the hard-coded test asset, although it usually *would* be if the repo config is read by ORT?

Reply:
I.e. this is a bit similar to the adjustments of the test asset in your last commit 398a6215568581f12d567ed362aab1ed3834d68f, correct?